src/architecture/_GeneralModuleFiles/sys_model.i

```diff
@@ -0,0 +1,36 @@
+%module(directors="1") sysModel
+%{
+   #include "sys_model.h"
+%}
+
+%pythoncode %{
+from Basilisk.architecture.swig_common_model import *
+%}
+%include "std_string.i"
+%include "swig_conly_data.i"
+%include "architecture/utilities/bskLogging.h"
+
+%feature("director") SysModel;
+%feature("pythonappend") SysModel::SysModel %{
+    self.__super_init_called__ = True%}
+%rename("_SysModel") SysModel;
+%include "sys_model.h"
+
+%pythoncode %{
+class SuperInitChecker(type):
+
+    def __call__(cls, *a, **kw):
+        rv = super(SuperInitChecker, cls).__call__(*a, **kw)
+        if not getattr(rv, "__super_init_called__", False):
+            error_msg = (
+               "Need to call parent __init__ in SysModel subclasses:\n"
+               f"class {cls.__name__}(sysModel.SysModel):\n"
+               "    def __init__(...):\n"
+               "        super().__init__()"
+            )
+            raise SyntaxError(error_msg)
+        return rv
+
+class SysModel(_SysModel, metaclass=SuperInitChecker):
+    bskLogger: BSKLogger = None
+%}
```

Comment:
:small_red_triangle_down: File needs a new line at the end. Notice how github tells you this with the little red symbol in the gutter on the left.  https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
Was not aware of this requirement. Will fix.